### PR TITLE
frontend: club selection

### LINKED ISSUE #68: Implement clubs selection view based on wireframe
https://www.figma.com/file/CTaCig2GBm6Tn6qvpA5nl32x/Pinpoint

### PR DESCRIPTION
**Pull request by Rahman2600**
:tickets: **Ticket(s)**: Closes #68 

---

## :construction_worker: Changes

Finished club selection view

### CODE REVIEW

File: frontend/src/components/ClubsSelection.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  }
+
+  // Mock loading of database object into component
+  componentDidMount() {
+    this.setState({ clubs: [{ name: 'UBC Launchpad 1', imagePath: '../assets/ubclaunchpad-logo.png' }, { name: 'UBC Launchpad 2', imagePath: '../assets/ubclaunchpad-logo.png' }] });
+  }
+
+  // TODO Replace unique key with database id of club
+  generateclubs() {
+<<<<<<< HEAD
```
Comment (leesw98) [inline frontend/src/components/ClubsSelection.js]: There is a merge thing here. You probably want to delete from here to line 21

File: frontend/src/styles/utils/flex.scss [hunk 2]
```diff
@@ -61,3 +61,7 @@
     &-end { align-content: flex-end }
   }
 }
+
+.csv {
```
Comment (leesw98) [inline frontend/src/styles/utils/flex.scss]: This class doesn't belong in this file. Perhaps spacing.scss

File: frontend/src/styles/utils/spacing.scss [hunk 3]
```diff
@@ -68,3 +73,9 @@ $spacing-xxl: 48px;
     &-xs { margin-bottom: $spacing-xs }
   }
 }
+
+.margin {
+  &-auto {display: flex;
```
Comment (bobheadxi) [inline frontend/src/styles/utils/spacing.scss]: take a look at the styling used elsewhere, and try to adhere to that - in this case, we should probably have `display` on a new line!

File: frontend/src/styles/utils/flex.scss (hunk 2 above)
Comment (bobheadxi) [reply frontend/src/styles/utils/flex.scss]: in fact, it doesnt look like this class is in use at all, so it can probably be removed

**Review by bobheadxi — COMMENTED** [review]
good stuff, a few things that need fixing but otherwise should be fine :)

**Comment by bobheadxi** [comment]
@Rahman2600 also, a note on commit messages (this applies to everyone in @ubclaunchpad/pinpoint as well!)

image

commit messages are for the sake of communicating your changes in a digestible manner, so that some day in the future if something needs to be reverted or if someone wants to work out why some code is the way it is, they have something to refer back to. [This guide](https://chris.beams.io/posts/git-commit/#seven-rules) gives a great intro to how to write good commit messages.

File: frontend/src/styles/utils/spacing.scss (hunk 3 above)
Comment (Rahman2600) [reply frontend/src/styles/utils/spacing.scss]: Do you mean the formatting?

Comment (bobheadxi) [reply frontend/src/styles/utils/spacing.scss]: yep! if you have `sass-lint` and the [VS code extension](https://marketplace.visualstudio.com/items?itemName=glen-84.sass-lint) for it installed, it'll tell you where there are issues as well (though unfortunately it cant fix them for you 😢 )

**Review by bwdmonkey — DISMISSED** [review]
fails to compile. removing `ready` tag

**Review by bobheadxi — APPROVED** [review]
Awesome! this has been sitting around for a while so i think we'll go ahead and merge this now 😁 

Thanks a lot @Rahman2600 !

**Pull request merged**